setup.py

```diff
@@ -130,6 +133,7 @@ def run(self):
         'doc': doc,
         'docker': docker,
         'druid': druid,
+        'gcs': gcs,
```

Comment:
Can you make this 'gcp' instead of 'gcs'? That way we can lump GCS, BQ, Datastore, etc libraries into it.